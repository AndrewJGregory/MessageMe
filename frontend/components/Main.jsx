import React from "react";
import Sidebar from "./Sidebar";
import MessageContainer from "./MessageContainer";
import MessageHeaderContainer from "./MessageHeaderContainer";
import MessageInputContainer from "./MessageInputContainer";

const Main = () => {
  return (
    <main className="main-container">
      <Sidebar />
      <div className="right-collection">
        <MessageHeaderContainer />
        <MessageContainer />
        <MessageInputContainer />
      </div>
    </main>
  );
};

export default Main;
