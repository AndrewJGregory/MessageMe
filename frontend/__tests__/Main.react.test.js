import React from "react";
import Main from "../components/Main";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Main username={"andrew"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
