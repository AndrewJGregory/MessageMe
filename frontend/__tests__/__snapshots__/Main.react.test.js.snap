// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  You're logged in as 
  andrew
  <button
    className="sign-out-btn clickable"
    onClick={undefined}
  >
    Sign Out
  </button>
  <div
    className="under-construction-info"
  >
    <p
      className="other-text"
    >
      This is currently under construction. While you're here, check out what else I've done:
    </p>
    <ul
      className="links"
    >
      <li>
        <a
          href="https://www.linkedin.com/in/andrewjgregoryajg/"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-linkedin"
          />
        </a>
      </li>
      <li>
        <a
          href="http://www.andrewjgregoryajg.com"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-globe"
          />
        </a>
      </li>
      <li>
        <a
          href="https://github.com/AndrewJGregory/"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-github"
          />
        </a>
      </li>
    </ul>
  </div>
  <div
    className="under-construction-img"
  />
</main>
`;
